braindecode/datautil/serialization.py

```diff
@@ -114,12 +116,33 @@ def _load_signals_and_description(path, preload, is_raw, ids_to_load=None):
 
 
 def _load_signals(fif_file, preload, is_raw):
```

Comment:
maybe we could add type hints when modifying this function? reading the code could be much easier then

Reply:
Maybe for another PR.